iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -1590,10 +1593,18 @@ public Map<String, String> getNativeProperties(org.apache.hadoop.hive.ql.metadat
   public boolean shouldOverwrite(org.apache.hadoop.hive.ql.metadata.Table mTable, String operationName) {
     String mode = null;
     String formatVersion = mTable.getTTable().getParameters().get(TableProperties.FORMAT_VERSION);
-    // As of now only delete mode is supported, for all others return false
-    if ("2".equals(formatVersion) && operationName.equalsIgnoreCase(Context.Operation.DELETE.toString())) {
-      mode = mTable.getTTable().getParameters()
-          .getOrDefault(TableProperties.DELETE_MODE, TableProperties.DELETE_MODE_DEFAULT);
+    // As of now only update & delete modes are supported, for all others return false
+    if ("2".equals(formatVersion)) {
+      switch (Context.Operation.valueOf(operationName)) {
```

Comment:
Ignore case is lost here. 
Should it be
```
Context.Operation.valueOf(operationName.toUpperCase())
```
?

Reply:
omg, thanks!